conf/poky/local.conf.sample

```diff
@@ -0,0 +1,241 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+ 
+# This file is your local configuration file and is where all local user settings
```

Comment:
Should this be included in a layer that only provides package recipes, no machine or distro?

Reply:
Splitting recipes and configs into separate layers is more design then technical solution.
It's possible to put all recipes which you need into single layer or split into many ones.

It's a good design when high level configs defining "project" to build (what machine, what distro and distro features, what image features and so on) are located into separate top layer.

Right now i don't know how many layers will be needed because even did not try to build prplmesh for RDKB or Yocto 3.1

I offer to postpone changes according your suggestion.
I've add separate task for that - #7